src/backends/ilp-quoter/index.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const _ = require('lodash')
+const request = require('co-request')
+const BigNumber = require('bignumber.js')
+const AssetsNotTradedError = 
+  require('../../errors/assets-not-traded-error')
+const NoAmountSpecifiedError = 
+  require('../../errors/no-amount-specified-error')
+const UnacceptableQuoterAmountError = 
+  require('../../errors/unacceptable-quoter-amount-error')
+const log = require('../../services/log')('ilpquote')
+const config = require('../../services/config')
+
+function lookupCurrencies (source_ledger, destination_ledger) {
+  for (let pair of config.get('tradingPairs').toJS()) {
+    if (pair[0].indexOf(source_ledger) === 4 &&
```

Comment:
I think it would be more clear to do something like `pair.split('@', 2)[1]` and have a separate method that parses the string format into a POJO along with a comment that shows an example like `USD@http://localhost:3000`